Router/Endpoint.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Endpoint.swift
+//  Pods
+//
+//  Created by Tadeáš Kříž on 6/6/15.
+//
+//
+
+import Foundation
+import Alamofire
+
+public protocol Endpoint {
+    typealias Input
+    typealias Output
+    
+    init(_ path: String)
```

Comment:
I would remove this initializer. If you want to use it, create new class with it. That way BaseEndpoint doesn't have to implement it.